SynLog.pas

```diff
@@ -5682,6 +5686,7 @@ procedure TSynLogFileView.AddInMemoryLine(const aNewLine: RawUTF8);
     tm: cardinal;
 begin
   tm := fThreadMax;
+  fLineLevelOffset:=0;

```

Comment:
this will break the expected behavior of this method, for sure

Reply:
it's buggy, the multithreaded Lazarus app just don't able to send log to viewer. Or viewer can't read it

Reply:
that way it works, the other **dont** . Fix it better, if you can. 
My goal is to make logger functional for Lazarus app. I **made** it functional. I don't have time to delve into spaghetti, which implements logger logic.

Reply:
https://stackoverflow.com/help/mcve is always better than carping I guess